pdns/Makefile.am

```diff
@@ -1143,6 +1166,7 @@ dnsgram_SOURCES = \
 	nsecrecords.cc \
 	qtype.cc \
 	rcpgenerator.cc rcpgenerator.hh \
+	svc-records.cc \
```

Comment:
supernit: somewhat-but-not-quite alphabetical sorting ;)

Reply:
I'll make a ticket that we need to sort them all :)